common/decisions/SRS_decisions.txt

```diff
@@ -1,12 +1,8 @@
 SRS_political_discussion = {
 	SRS_elections_timer = {
 		icon = eng_propaganda_campaigns
-		activation = { 
```

Comment:
Why all these changes? Please consult @Jas-PY

Reply:
All of these checks are already checked in the category, so they are not needed here.